presto-orc/src/test/java/com/facebook/presto/orc/TestingColumnHandle.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Ouch, that's my fault. `TestingColumnHandle` from presto-spi lies in `test` scope.
Please disregard my comment then.